test/e2e/apps/rc.go

```diff
@@ -558,3 +563,56 @@ func updateReplicationControllerWithRetries(c clientset.Interface, namespace, na
 	}
 	return rc, pollErr
 }
+
+type rcPatchFunc func(rc *v1.ReplicationController) ([]byte, error)
+
+func patchRCOrDie(cs kubernetes.Interface, dc dynamic.Interface, ns string, name string, f rcPatchFunc, subresources ...string) (updated *v1.ReplicationController) {
+	err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
+		old := getRCStatusOrDie(dc, ns, name)
+		patchBytes, err := f(old)
+		framework.ExpectNoError(err, "Failed to run ReplicationController patch function for resource %s", name)
```

Comment:
please make all error annotations to start lowercase
`the ReplicationController....`
[1]